docs/content/latest/api/ycql/batch.md

```diff
@@ -0,0 +1,173 @@
+---
+title: BATCH requests [YCQL]
+headerTitle: BATCH
+linkTitle: BATCH
+summary: Execute multiple DML in 1 request
+description: Use batch to update multiple rows in 1 request.
+menu:
+  latest:
+    parent: api-cassandra
+    weight: 19991
+aliases:
+  - /latest/api/ycql/batch2
+isTocNested: true
+showAsideToc: true
+---
+
+Batch operations let you send multiple operations in a single RPC call to the database. The larger the batch size, 
+the higher the latency for the entire batch. Although the latency for the entire batch of operations is higher than the latency of any single operation, 
+the throughput of the batch of operations is much higher.
+
+## Example in JAVA
+
+In order to perform a batch insert operation in Java, first create a BatchStatement object. 
+Next add the desired number of prepared and bound insert statements to it. 
+Finally, execute the batch object. This is shown below.
+
+```java
+// Create a batch statement object.
+BatchStatement batch = new BatchStatement();
+
+// Create a prepared statement object to add to the batch.
+PreparedStatement insert = client.prepare("INSERT INTO table (k, v) VALUES (?, ?);");
+
+// Bind values to the prepared statement and add them to the batch.
+for (...) {
+  batch.add(insert.bind( ... <values for bind variables> ... ));
+}
+
+// Execute the batch operation.
+ResultSet resultSet = client.execute(batch);
+
+```
+## Example in Python using RETURNS AS STATUS
+
+An example using Python client and RETURNS AS STATUS clause:
+
+```cassandra
```

Comment:
Is it cassandra or cassandraql? A lot of the other snippets just use `sql` because there used to be no highlighting declared for cql.